includes/discovery/libvirt-vminfo.inc.php

```diff
@@ -66,10 +66,34 @@
                     $vmwVmDisplayName = $xml->name;
                     $vmwVmGuestOS     = '';
                     // libvirt does not supply this
-                    $vmwVmMemSize = ($xml->currentMemory / 1024);
-                    exec($config['virsh'].' -c '.$uri.' domstate '.$dom_id, $vm_state);
+                    exec($config['virsh'].' -rc '.$uri.' domstate '.$dom_id, $vm_state);
                     $vmwVmState = ucfirst($vm_state[0]);
-                    $vmwVmCpus  = $xml->vcpu;
+
+                    $vmwVmCpus = null;
+                    $vmwVmMemSize = null;
+                    if ($xml->xpath("//sysinfo/system/entry[@name='manufacturer'][.='oVirt']")) {
+                        d_echo("Parsing information for $vmwVmDisplayName with oVirt quirx");
```

Comment:
I'm pretty sure oVirt is following libvirt XML format (see https://libvirt.org/formatdomain.html and specifically https://libvirt.org/formatdomain.html#elementsMemoryAllocation - I there unsure why the comment of 'quirkx' is relevant here. How does the XML look like when running from Virt-Manager?